src/app/core/services/course-service/course.service.ts

```diff
@@ -21,7 +21,8 @@ export class CourseService {
      * method that adds course
      */
     public addCourse(course: ICourse): void {
-        this.courses = [...this.courses, course];
+        const courseWithID: ICourse = {...course, id: this.courses.length};
```

Comment:
this can cause id duplicating after removing course

Reply:
this is temp solution until we have actual backend that will generate ids